pkgs/by-name/bp/bpfilter/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  bison,
+  cmake,
+  cmocka,
+  doxygen,
+  fetchFromGitHub,
+  flex,
+  lcov,
+  lib,
+  libbpf,
+  libnl,
+  nix-update-script,
+  pkg-config,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bpfilter";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = "bpfilter";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-894SxGhXxkxYD11abl8HfjYyhGFpXGx1JpCsF35/jAw=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    cmake
+    cmocka
+    doxygen
+    flex
+    lcov
+    pkg-config
+    python3.pkgs.sphinx
+  ];
+
+  buildInputs = [
+    libbpf
+    libnl
+  ];
+
+  cmakeFlags = [
+    "-DNO_TESTS=1" # tries to clone nftables
+    "-DNO_CHECKS=1"
+    "-DNO_BENCHMARKS=1"
+  ];
+
+  # required to use interface name as format string
+  env.NIX_CFLAGS_COMPILE = "-Wno-format-security";
```

Comment:
Would `hardeningDisable = [ "format" ]` be a viable alternative?

Reply:
no longer needed with 0.2.1!